pkgs/development/libraries/avahi/default.nix

```diff
@@ -9,6 +9,11 @@
 
 assert qt4Support -> qt4 != null;
 
+let
+  # despite the configure script claiming it supports $PKG_CONFIG, it doesnt respect it
+  pkgconfig-helper = writeShellScriptBin "pkg-config" ''exec $PKG_CONFIG "$@"'';
```

Comment:
Shouldn't this be:

```nix
pkgconfig-helper = writeShellScriptBin "${buildPackages.pkg-config}/bin/pkg-config" ''exec $PKG_CONFIG "$@"'';
```

As this ends up executing in loop re-executing itself in non-cross uses?